Divided approach (#482)
lethe-cfd/lethe: doc/source/parameters/unresolved_cfd-dem/void_fraction.rst

   set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
   end
     

> If the ``mode`` chosen is ``pcm``, then the void fraction is calculated using the Particle Centered Method. If it is set to ``qcm``, then the void fraction is calculated using the Quadrature Centered Method. If it is set to ``spm``, then the void fraction is calculated using the satellite point method (divided approach). In these methods, the remaining parameters are required:

Reviewer: What do you think of naming it satellite point instead of spm inside the prm file? I think spm is not as usual.

Author: In the source code they are all named fully. But I think it is cleaner and easier for the user to insert the names as abbreviations in the prm file. All abbreviations are explained in the documentation

> If the ``mode`` chosen is ``pcm``, then the void fraction is calculated using the Particle Centered Method. If it is set to ``qcm``, then the void fraction is calculated using the Quadrature Centered Method. If it is set to ``spm``, then the void fraction is calculated using the satellite point method (divided approach). In these methods, the remaining parameters are required:

Reviewer: I'd do the same for "qcm" and use something as "quadrature" instead.

* The ``dem_file_name`` parameter specifies the prefix of the dem files that must be read.
* The ``l2 smoothing factor`` is a smoothing length used for smoothing the L2 projection of the void fraction to avoid sharp discontinuities which can lead to instabilities in the simulation.
* The ``l2 lower bound`` and ``l2 upper bound`` are the minimum and maximum values around which the void fraction is bounded. This is important especially for upper bounds as the void fraction can sometimes slightly exceed a value of 1 when projected.
* The ``bound void fraction`` parameter determines whether or not to bound the void fraction between the lower and upper bounds specified in the previous two parameters. As the void fraction is calculated and then projected using L2 projection, it can sometimes exceeds the maximum value of 1. In order to prevent this, we use an active set method to bound the void fraction.

> * The ``particle refinement factor`` is only required for the ``spm``, to determine how many particles we want to divide our particle. By default, it is set to 0 refinements, and results in no refinement of the original meshed particle (division into 7 particles). Every additional refinement results in a :math:`2^{dim}` times more particles.

Reviewer 2: The first sentence is unclear.

> * The ``particle refinement factor`` is only required for the ``spm``, to determine how many particles we want to divide our particle. By default, it is set to 0 refinements, and results in no refinement of the original meshed particle (division into 7 particles). Every additional refinement results in a :math:`2^{dim}` times more particles.

Author: done

> * The ``number of particle refinement `` is only required for the ``spm``, to determine how many particles we want to divide our particle. By default, it is set to 0 refinements, and results in no refinement of the original meshed particle (division into 7 particles). The more refinements we use, the more particles we will divide our original particle.

Reviewer: Also here, you could call this parameter "particle refinement factor". "Number of particle refinements" gives the idea that you are setting the number of satellite points.

Author: I agree with you. The name should be different. I will change it

> * The ``number of particle refinement `` is only required for the ``spm``, to determine how many particles we want to divide our particle. By default, it is set to 0 refinements, and results in no refinement of the original meshed particle (division into 7 particles). The more refinements we use, the more particles we will divide our original particle.

Reviewer: ```suggestion
* The ``number of particle refinement`` is only required for the ``spm``, to determine how many particles we want to divide our particle. By default, if no ``number of particle refinement`` is provided, particles are divided into 7 satellite points (``number of particle refinement = 0``).  The more refinements we use, the more particles we will divide our original particle.
```

Reviewer: Additionally I'd provide more details regarding the progression of the amount of satellite points with the parameter

Author: Good idea. Will work on it as well
